src/routes/timeTaskRoute.js

```diff
@@ -0,0 +1,9 @@
+const timeTaskCtrl = require('../controllers/timeTaskCtrl');
+
+module.exports = (app) => {
+  app.post('/time_task', timeTaskCtrl.createTimeTask);
+  app.get('/time_tasks', timeTaskCtrl.getTimeTasks);
+  app.get('/time_task/:time_task_id', timeTaskCtrl.getOneTimeTask);
+  app.put('/time_task/:time_task_id', timeTaskCtrl.updateTimeTask);
+  app.delete('/time_task/:time_task_id', timeTaskCtrl.deleteTimeTask);
```

Comment:
In my vision of things, I would have seen more roads of the following form: 

```suggestion
  app.post('/task/:task_id/start', timeTaskCtrl.startTimeTask);
  app.post('/task/:task_id/start', timeTaskCtrl.endTimeTask);
  app.get('/task/:task_id/timers', timeTaskCtrl.getTimeTasks);
  app.get('/task/:task_id/timer/:time_task_id', timeTaskCtrl.getOneTimeTask);
  app.put('/task/:task_id/timer/:time_task_id', timeTaskCtrl.updateTimeTask);
  app.delete('/task/:task_id/timer/:time_task_id', timeTaskCtrl.deleteTimeTask);
```

So we don't need to pass the body, the start_date is automatic when we make a start and the end_date is automatically generated on the controller side to add it if it is already start.

What do you think about it ? @kilianpichard @BrystoQ

Reply:
Indeed, I agree with this proposal.
It will make more sense for the understanding of the routes